docs/source/calibration.md

```diff
@@ -0,0 +1,68 @@
+# Calibration Algorithm in Quantization
+
+1. [Introduction](#introduction)
+2. [Calibration Algorithm](#calibration-algorithm)
+3. [Supported Framework Matrix](#supported-framework-matrix)
+
+## Introduction
+
+Quantization proves beneficial in terms of reducing the memory and computational requirements of the model. Uniform quantization transforms the input value $x ∈ [β, α]$ to lie within $[−2^{b−1}, 2^{b−1} − 1]$, where $[β, α]$ is the range of real values chosen for quantization and $b$ is the bit-width of the signed integer representation. Calibration is the process of determining the $α$ and $β$ for model weights and activations. Refer to this [link](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md#quantization-fundamentals) for more quantization fundamentals
+
+## Calibration Algorithm
+
+There are three main methods of Calibration:

```

Comment:
How about changing to `Currently, INC supports three popular calibration algorithms:`? as there are others algo, such as MovingAverage?